### PR TITLE
Update IndexCreator

### PR DESCRIPTION
**Pull request by damien-carcel**
Minor typo, PHPDoc and useless variables fixes

| Q                 | A
| ----------------- | ---
| Added Specs       | N/A
| Added Behats      | N/A
| Travis CI is ok   | Running
| Changelog updated | No

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/IndexCreator.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 use Pim\Component\Catalog\Model\ChannelInterface;
 use Pim\Component\Catalog\Model\CurrencyInterface;
 use Pim\Component\Catalog\Model\LocaleInterface;
+use Pim\Bundle\CatalogBundle\Repository\AttributeRepositoryInterface;
```
Comment (phaseinducer) [inline src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/IndexCreator.php]: Wrong order, CS fixer breaks here

**Comment by phaseinducer** [comment]
GTM after fix

File: src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/IndexCreator.php (hunk 1 above)
Comment (damien-carcel) [inline src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/IndexCreator.php]: In fact, right order but wrong namespace, I report a fix from 1.4 and didn't realize these interface were now in Component and not Bundle anymore.

**Comment by skeleton** [comment]
GTM

**Pull request merged**